packages/core/src/utils/test-utils.ts

```diff
@@ -94,6 +94,7 @@ export const createContextWithRouteParameters = (
 
   return {
     ...ctx,
+    path: ctx.path,
```

Comment:
this LOC seems does not change anything.

Reply:
without that line, the tests don’t behave as expected and fail.

Reply:
should be because the function checks for the value of ctx.path, but that line makes the tests pass as they should. i tested further by intentionally putting expect statements that should FAIL, to check if the LOC somehow makes all tests pass, but they failed as expected.